target/linux/ath79/dts/qca9533_dlink_dap-13xx.dtsi

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	aliases {
+		led-boot = &led_power_green;
```

Comment:
Based on the manual the red color indicates booting.

Reply:
During u-boot execution, the status LED is red, then goes off for about two seconds before it starts the typical (green) OpenWRT boot flashing pattern. This is what I am used to from other OpenWRT devices, though honestly I'm not even exactly sure what the led-boot alias is for; also, would we need gpio-hog or something to get rid of the short off period?

Reply:
what's written in the aliases section is executed here: https://github.com/openwrt/openwrt/blob/master/package/base-files/files/etc/diag.sh

So, this will be OpenWrt's boot blinking pattern. To be honest, I don't care so much about whether this LED blinks red or green then, but the "proper" way would be to do it the same way as the vendor (if that's possible). As a bonus, you will then have different colors for boot and failsafe, for example ...